### PR TITLE
Add migrations health check

### PR DESCRIPTION
**Pull request by timohund**
This commit:

* Add's a healthcheck to test if all pending migrations have been exected

Fixes: #37

### CODE REVIEW

File: src/Checks/MigrationUpToDateHealthCheck.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    protected $migrator;
+
+    /**
+     * @return Status
+     */
+    public function status()
+    {
+        try {
+            if (!$this->isDatabaseUpToDate()) {
+                return $this->problem('Not all migrations have been executed', []);
```
Comment (tylerwoonton) [inline src/Checks/MigrationUpToDateHealthCheck.php:25]: I think it'd be nice to return the migrations which have not been ran in the problem context here.

**Review by tylerwoonton — CHANGES_REQUESTED** [review]
Hi @timohund, thanks for contributing.

The migrations check looks good and I think it would be useful. 

I've just raised a concern about the empty problem context in this review. I'll do a full test once that's been addressed - it shouldn't be too difficult. 

Feel free to reach out if you have any issues.

File: src/Checks/MigrationUpToDateHealthCheck.php (hunk 1 above)
Comment (timohund) [reply src/Checks/MigrationUpToDateHealthCheck.php:25]: @tylerwoonton thx for your usefull feedback, i will add that and update the pr

**Review by tylerwoonton — APPROVED** [review]
Tested successfully with both `OK` and `PROBLEM` statuses using Laravel 5.5 and Laravel 8 with MySQL.

@timohund This looks great - I'll go ahead and get this merged in.

**Comment by tylerwoonton** [comment]
Tagged at `v1.7.0`.

https://github.com/ukfast/laravel-health-check/releases/tag/v1.7.0

**Pull request merged**